namespace/id.go

```diff
@@ -27,12 +27,7 @@ func (nid ID) Size() IDSize {
 	return IDSize(len(nid))
 }
 
-// String stringifies the nid.
+// String returns the hexadecimal encoding of the nid.
```

Comment:
[optional] Thank you for implementing the additional test for `string(nid)`. To ensure full understanding, I recommend we include a remark at this point to clarify that `string(nid)` is not equivalent to `nid.String()`. This clarification is important, as these two can easily be mistaken for each other.

Reply:
Resolved by https://github.com/celestiaorg/nmt/pull/202/commits/d71f15fbdb7189d8fd39d54bfc28d7f11296f1bb